src/pages/_app.page.tsx

```diff
@@ -1,4 +1,5 @@
 import "@/styles/globals.css";
+import '@chainverse/os/dist/chainverse-os.css';
```

Comment:
When I test this branch locally I get the following error related to this line 
![image](https://user-images.githubusercontent.com/5624429/180297576-b3f4863d-d082-4416-96d8-a6f9a3f9c922.png)

Reply:
Disregard - I had to run `npm i` to reinstall dependencies